lib/engine/game/g_1822_pnw/game.rb

```diff
@@ -82,7 +82,7 @@ def price_movement_chart
         PRIVATE_CLOSE_AFTER_PASS = %w[P11].freeze
         PRIVATE_PHASE_REVENUE = %w[].freeze # Stub for 1822 specific code
 
-        IMPASSABLE_HEX_COLORS = %i[gray red].freeze
+        IMPASSABLE_HEX_COLORS = %i[gray red blue brown].freeze
```

Comment:
Is this compatible with the bridge builder and ports?

Reply:
Yes.  Currently IMPASSABLE_HEX_COLORS is only used in connect_hexes and lay_tile to set up neighbors, it's not used as a restriction to placing a tile.